app/views/people/show.html.haml

```diff
@@ -20,6 +20,10 @@
   %dl
     %dt Role w systemie
     %dd= @person.roles.join(", ")
+  %dl
+    %dt Więcej informacji
+    %dd= link_to "Google", search_person_path(@person.name, @person.surname), class: %w{btn btn-primary btn-sm}
```

Comment:
Wystarczy `search_person_path(@person)`. Przydałoby się to otworzyć w osobnej karcie (`target: "_blank"`)